extract furthest point in terms of opening angle in postfit

philippeller opened the pull request:
Attempt at adding the angular uncertainty to the "furthest_point" estimators

atfienberg commented:
The logic looks good to me! I'm interested to see how this performs. Did you manage to test it at all yet?

Could you put the logic to calculate the angular distance into its own function just to keep the `postfit` function a little bit tidier? It doesn't really do any calculations right now; it only calls the specific postfit subroutines and then collects their results. Also, in your new function, I recommend adding a parameter to select a different delta LLH cut than the default of 15 that we use to calculate the mean and stddev.  For example, the existing `furthest_point` function uses a default delta LLH cut of 2.

philippeller commented:
Cool, I have a addressed your comments now. And yes, the code is tested that it produces meaningful results, but i have not yet assessed the quality of this new uncertainty.

atfienberg commented on freedom/reco/postfit.py:
```diff
@@ -256,6 +290,8 @@ def postfit(all_pts, par_names=PAR_NAMES, llh_cut=DELTA_LLH_CUT):
     llh_cut : float, default 15
         postfit routines only consider samples within llh_cut llh of the best sample
+    aux_names : list
+        any auxillary quantities to be extracted
```
"auxiliary"

atfienberg commented on freedom/reco/postfit.py:
```diff
@@ -280,6 +316,14 @@ def postfit(all_pts, par_names=PAR_NAMES, llh_cut=DELTA_LLH_CUT):
+            zen = cut_pts[:, par_names.index("zenith")]
+            furthest_points.append(furthest_angular_distance(az, zen, cut_llhs, above_min=2))
+        else:
+            raise ValueError("Unknown auxillary quantity `%s`"%name)
```
"auxiliary"

atfienberg reviewed (approved):
Looks great to me. Only thing is that I noticed there are a couple spelling errors in the comments / exception messages.

philippeller commented:
ok, I did a `:%s/auxillary/auxiliary/g`
